### PR TITLE
Openstack controller - Dashboard updated

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
# Summary

Ticket URL: https://issues.newrelic.com/browse/NR-167462
Description: Updated the doc URL and logo URL in the Openstack controller dashboard.

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/openstack-controller/openstack-controller.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
...
+            "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details?state=96ef40d3-95de-189c-b44b-845e5ed3663b).\n\nInstrument Openstack Controller with New Relic using the [documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration/).\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=OpenstackController&entry.358368110=https://onenr.io/0EjONkYNDR6) here and let us know how we can improve it for you."
```
Comment (rahulbasu-nr) [inline dashboards/openstack-controller/openstack-controller.json:122]: Please review this link properly.

Comment (rahulbasu-nr) [inline dashboards/openstack-controller/openstack-controller.json:122]:
```suggestion
            "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details?state=96ef40d3-95de-189c-b44b-845e5ed3663b).\n\nInstrument Openstack Controller with New Relic using the [documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/openstack-controller-integration/).\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=OpenstackController) here and let us know how we can improve it for you."
```

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Please make the changes.

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Pull request merged**